images/saige-qtl/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-r:1.4
+
+ENV MAMBA_ROOT_PREFIX /root/micromamba
+
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
+RUN micromamba install -y --prefix $MAMBA_ROOT_PREFIX \
+        -c cpg -c bioconda -c conda-forge \
+        r-essentials r-base=4.1.2 python=3.10 \
+        anaconda cmake \
```

Comment:
That's probably already installed through the base as well, in addition to a few of the things below?

Reply:
yes good point, i can try and remove one by one to check? Or is there a better way to make sure?

Reply:
I don't know a great way for conda unfortunately. The other approach is to build it up in reverse: only start with what you know you truly need and then add more packages when you encounter remaining dependency problems.

Reply:
I am installing everything recommended [here](https://github.com/saigegit/SAIGE/blob/main/conda_env/createCondaEnvSAIGE_steps.txt) - linked from https://saigegit.github.io/SAIGE-doc/docs/Installation_conda.html

Reply:
Ah, got it! `anaconda` is not a package though, it's the channel (the parameter after `-c`).

Reply:
oops my bad! moved up now